nautobot/extras/models/metadata.py

```diff
@@ -221,20 +221,14 @@ class Meta:
     @property
     def value(self):
         if self.metadata_type.data_type == MetadataTypeDataTypeChoices.TYPE_CONTACT_TEAM:
-            return self.contact or self.team
+            return None
         else:
             return self._value
```

Comment:
This kinda defeats the purpose of shadowing the `_value` field at all, doesn't it though? Should we just change the db field back to `value` and remove the need for the `@property` at all?